README.md

```diff
@@ -127,10 +127,10 @@ simulate an install
 -------------------
 ### skip-deploy
 * **BitOps Property:** `skip-deploy`
-* **Environment Variable:** `SKIP_DEPLOY`
-* **default:** `""`
-
-will skip helm execution
+* **Environment Variable:** `HELM_SKIP_DEPLOY`
+* **default:** `none`
+* **Required:** `false`
+* **Description:** If set to true, regardless of the stack-action, deployment actions will be skipped.
```

Comment:
Looks like it's not `true`, but `True` upper-case per code here: 
https://github.com/bitops-plugins/helm/blob/b77f2968d55e5687cc73769b605a604cb453e028/scripts/helm_handle_chart.sh#L18-L22

Sounds like a bug? I see we use `true` lowercase for other plugins.